e2e_test/batch/functions/array_concat.slt.part

```diff
@@ -1,98 +1,3 @@
-query T
```

Comment:
These tests are duplicates with docslt (actually introduced before we had docslt)